frontend/resources/styles/main/partials/dashboard-team.scss

```diff
@@ -19,12 +19,30 @@
 
   .custom-input {
     width: 314px;
+    height: 14px;
+    font-size: 14px;
     margin-right: 10px;
+
+    input {
+      padding: 0;
+
+      &.empty {
+        margin-top: 10px;
+        &::placeholder {
+          color: #b1b2b5;
```

Comment:
Try to use color variables. They are define in this file frontend/resources/styles/common/dependencies/colors.scss
That color your using here would be `color: $color-gray-20;`

Reply:
fixed